experiments/chaos/openebs_volume_replica_failure/run_litmus_test.yml

```diff
@@ -1,3 +1,12 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: target-failure
```

Comment:
@sakshamkatiyar change the name into replica-failure

Reply:
Done